.github/workflows/release.yml

```diff
@@ -66,6 +71,11 @@ jobs:
           -H "Content-Type: $(file -b --mime-type build/lavad)" \
           --data-binary @build/lavad \
           $(echo '${{ github.event.release.upload_url }}' | sed 's/{?name,label}/?name=lavad-${{ github.event.release.tag_name }}-linux-amd64/g')
+
+          #Upload to binary registry
+          aws s3 cp build/lavad s3://network-upgrades-zarak/"${{ github.event.release.tag_name }}"/linux-amd64/lavad
+          #Latest -- We need to define what is the latest for us
+          # aws s3 cp build/lavad s3://network-upgrades-zarak/latest/linux-amd64/lavad
```

Comment:
We need to define what we consider as "latest" is the latests on pre-release or the one stable?

This automation is triggered on modification of any Release entry, if we consider latest being the higher tag number, I'll be adding that condition.